### PR TITLE
fix pushing tags / yarn upgrade 10 april 2017

### PR DESCRIPTION
**Pull request by mattkime**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,23 @@
 sudo: required
 language: node_js
 node_js:
-  - "6.1"
+  - "7.8"
+cache:
```
Comment (mattkime) [inline .travis.yml:5]: to match other repos. faster builds.

File: package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "lint": "yarn run lint:base -- --fix",
     "lint:base": "eslint . --ext .js, .jsx",
     "modules:clean": "rm -rf node_modules",
-    "prepublish": "yarn run build",
+    "install": "yarn run build",
```
Comment (mattkime) [inline package.json:13]: for npm v5 compatibility. prepublish will no longer run automatically once v5 rolls out.

**Comment by mattkime** [comment]
merging since this is effectively preventing new revisions on npm. happy to revisit any changes.

File: .travis.yml [hunk 3]
```diff
@@ -22,11 +27,13 @@ addons:
       - g++-4.8
 
 before_install:
-    - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" > .npmrc
+  - openssl aes-256-cbc -K $encrypted_e09404c5d59a_key -iv $encrypted_e09404c5d59a_iv -in '.travis/github_deploy_key.enc' -out github_deploy_key -d
```
Comment (mmcgahan) [inline .travis.yml:30]: line comments would be really helpful here to give future devs an indication of how to work with this

**Pull request merged**